utils/utils.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import "os/exec"
+
+func ExecuteCommand(cmd string) string {
```

Comment:
Let's return `(string, error)` here. We shouldn't only return a `string` in an error condition, and I'd also want to see why `cmd` failed. This function will just eat that error instead.

A somewhat unavoidable pattern in Go is the `_, err := func(); if err != nil {}` pattern. I think it only makes sense to check for an error when calling `utils.ExecuteCommand()` as well. I still see utility in this helper function though, and it'll no doubt expand as we run more complex commands in the future.